eppoclient/initclient.go

```diff
@@ -4,7 +4,7 @@ package eppoclient
 
 import "net/http"
 
-var __version__ = "2.0.0"
+var __version__ = "3.0.0"
```

Comment:
do you see any reason to publish the package as `eppoclient/v3` ?

Reply:
We should bump the major version as public interface changed. Good catch on updating package name to `eppoclient/v3` — I forgot to update it in the mod file